c7n/filters/iamaccess.py

```diff
@@ -343,3 +343,116 @@ def __call__(self, r):
         if violations:
             r[self.annotation_key] = violations
             return True
+
+
+class PolicyStatementFilter(Filter):
+    """  Check a resource for a set of policy statements.
+    """
+    POLICY_ELEMENTS = [
+        'Sid', 'Effect', 'Principal', 'NotPrincipal', 'Action',
+        'NotAction', 'Resource', 'NotResource', 'Condition']
+    schema = type_schema(
```

Comment:
this class should have schema_alias = True on it

Reply:
added in d015a99